.jenkins/build_docker_images.sh

```diff
@@ -3,7 +3,7 @@
 # Set variable to allow parallel building in the Docker image creation
 export DOCKER_BUILDKIT=1
 export BUILDKIT_PROGRESS=plain
-targets="build build_serialize"
+targets="build build_serialize_gt4py_dev"
```

Comment:
@SpiderMonkey1975 Which is the correct target here, `build_serialize` or `build_serialize_gt4py_dev`?

Reply:
It depends on what these images are used for. If these are for running the fortran consistency tests on Piz Daint, then it should be build_serialize, if for making an image to serialize data on Piz Daint (which we don't do), it'd be build_serialize_gt4py_dev

Reply:
I have been using build_serialize.  build_serialize_gt4py_dev is a new option that I have never used or know what it does.

Reply:
you'll need to use it for any future fortran comparison runs, as the changes made when that flag is used are what our python model implements